documentation/migrating/05-endpoints.md

```diff
@@ -5,3 +5,5 @@ title: Endpoints
 In Sapper, 'server routes' — now referred to as [endpoints](/docs#routing-endpoints) — received the `req` and `res` objects exposed by Node's `http` module (or the augmented versions provided by frameworks like Polka and Express).
 
 SvelteKit is designed to be agnostic as to where the app is running — it could be running on a Node server, but could equally be running on a serverless platform or in a Cloudflare Worker. For that reason, you no longer interact directly with `req` and `res`. Your endpoints will need to be updated to match the new signature.
+
+To support this environment-agnostic behavior, `fetch` is now available in the global context, so you don't need to import `node-fetch`, `cross-fetch`, or similar server-side fetch implementations in order to use it.
```

Comment:
The 'now' here sounds a little odd, like we're writing a changelog or migration guide.

Reply:
Oh. Whoops. Guess what, this _is_ part of the migration guide. I no longer have any issue with the use of the word 'now'.